utils/get_default_params.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+import warnings
+
+
+def get_default_params(path):
+    """
+    Reads the default_params json file in the directory passed in path
```

Comment:
Please use google style doctstrings (with Args and Returns) when you describe the arguments and returns

Reply:
Done